apps/docs/libraries/ts/sdk/keys/verify.mdx

```diff
@@ -97,16 +93,43 @@ Example:
 
 </ResponseField>
 
-    <ResponseField name="remaining" type="int">
+  <ResponseField name="remaining" type="int">
     Shows how many more times this key may be verified before being invalidated.
     Only applies to keys where you have set a `remaining` count.
     </ResponseField>
-<ResponseField name="enabled" type="bool" required>
+
+<ResponseField name="code" type="string" >
+  If the key is invalid this field will be set to the reason why it is invalid.
+
+  Possible values are:
+  - `NOT_FOUND` - The key does not exist
+  - `FORBIDDEN` - You are not allowed to verify this key. For example because of additional security
+  checks like `IP whitelists`
+  - `USAGE_EXCEEDED` - The key has been used up and is no longer valid
+  - `RATE_LIMITED` - The verification has been blocked due to ratelimiting
+  - `DISABLED` - the key is disabled
+</ResponseField>
+
+ 
+<ResponseField name="enabled" type="bool">
     Shows if the key is enabled or disabled.
-    </ResponseField>
+</ResponseField>
+
+
+<ResponseField name="permissions" type="string[]">
+   A list of all the permissions this key is connected to.
+</ResponseField>
+
+<ResponseField name="environment" type="string">
+ The environment of the key, this is what what you set when you crated the key
+</ResponseField>
+
+<ResponseField name="identity" type="string">
```

Comment:
This isn't a string but an object actually
see https://github.com/unkeyed/unkey/blob/81913daf6706b13d1e85c3e51df7ee4e3d8c5762/apps/api/src/routes/v1_keys_verifyKey.ts#L260

Reply:
Copy paste error,  fixed now.